src/shapes/group.class.js

```diff
@@ -349,7 +352,7 @@
       this._setObjectPosition(object);
 
       object.setCoords();
-      object.hasControls = object.__origHasControls;
+      object.hasControls = !!object.__origHasControls;
```

Comment:
ok i learned something new!
I think we **have to** use _updateObjectsCoords... i need time to test it.